content/c/concepts/pointers/pointers.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: "Pointers" 
+Subjects:
+  - "Code Foundations"
+  - "Computer Science"
+Tags:
+  - "Pointers"
+  - "Memory"
+  - "Variables"
+  - "Arrays"
+CatalogContent:
+  - "learn-c-plus-plus"
+  - "paths/computer-science"
+---
+
+A pointer is a [variable](https://www.codecademy.com/resources/docs/c/variables) that stores a memory address, which typically represents the location of another variable.
+
+Pointers are very useful in that they allow the efficient creation and manipulation of complex data structures.
+
+## Understanding Pointers
+
+Data is stored in a computer's [memory](https://www.codecademy.com/resources/docs/general/what-is-ram). A variable represents a specific location within that memory, and consists of three parts: an identifier (name), a value, and an *address*. The address is a value that describes *where* in memory a variable is located.
+
+To understand pointers, it may be helpful to think of variables as boxes on a shelf. Each box has a label (the identifier), a unique serial number that allows one to locate it (the address), and possibly something inside of it (the value). Following this analogy, a pointer can be described as a box that *contains* the serial number of another box within it.
+
+## Declaring and Assigning a Pointer
+
+A pointer is declared similarly to how a variable is - by specifying its type, identifier, and value. However, an asterisk (`*`) character is inserted in front of the identifier.
+
+```pseudo
+type *name = value;
+```
+
+In order to obtain the address of a variable, the identifier is prefixed with an ampersand (`&`) symbol. This is known as the *address operator*.
```

Comment:
I know we usually use italics for keywords in Codecademy lessons, let's not use italics for keywords for Codecademy Docs.